apps/shelter-web/src/app/layout/footer.tsx

```diff
@@ -0,0 +1,29 @@
+import { ReactElement } from 'react';
+
+type IParams = {
+  className?: string;
+};
+
+export function Footer(props: IParams): ReactElement {
+  const { className = '' } = props;
+
+  const parentCss = [
+    className,
+    'w-full',
+    'flex',
+    'flex-col',
+    'min-h-52',
+    'py-12',
+    'text-white',
+  ].join(' ');
```

Comment:
I typically do this to keep classes cleaner (when it's more than a few), though probably should pull in `twMerge`